bpf/lib/common.h

```diff
@@ -880,9 +880,8 @@ struct ct_entry {
 	      proxy_redirect:1, /* Connection is redirected to a proxy */
 	      dsr:1,
 	      from_l7lb:1, /* Connection is originated from an L7 LB proxy */
-	      auth_required:1,
```

Comment:
shouldn't this be changed to ` reserved:1,`? Otherwise the `from_tunnel` will be in the wrong place

Reply:
there is no release with `from_tunnel` yet so I thought that would not matter?